src/main/java/ee/tuleva/onboarding/auth/jwt/JwtTokenUtil.java

```diff
@@ -60,14 +59,18 @@ public Person getPersonFromToken(String token) {
   }
 
   private Claims getAllClaimsFromToken(String token) {
+    // When token is expired, this throws ExpiredJwtException, which is handled in
```

Comment:
I'd prefer `ExpiredTokenException` - that wouldn't leak implementation details (that we're currently using a token in JWT format).